digiwf-engine/digiwf-engine-service/src/main/resources/application.yml

```
@@ -185,6 +185,16 @@ digiwf:
   s3service:
     topic: 'dwf-s3-${DIGIWF_ENV}'
     httpAPI: ${DIGIWF_S3_HTTPAPI:http://localhost:8086}
+  prometheus:
+    process-engine:
+      update-interval: 3000
```

```suggestion
      update-interval: 30000
```

Sind aktuell 3000ms -> 3s wenn ich die Doku richtig gelesen habe

---

das ist die echte Engine. Ok aus meiner Sicht.